ch.sbb.scion.rcp.workbench/src/ch/sbb/scion/rcp/workbench/view/ViewIntentInterceptor.java

```diff
@@ -62,43 +62,55 @@ public boolean handle(IntentMessage<Map<String, ?>> intentMessage) throws PartIn
     return false;
   }
 
-  private void openMicrofrontendEditor(IntentMessage<Map<String, ?>> intentMessage) throws PartInitException {
+  private void openMicrofrontendEditor(final IntentMessage<Map<String, ?>> intentMessage) throws PartInitException {
     var activePage = getActivePage();
     var capability = intentMessage.capability;
     var intent = intentMessage.intent;
-    var target = Optional.ofNullable(intentMessage.body).map(body -> (String) body.get("target")).orElse("default");
+    var target = Optional.ofNullable(intentMessage.body).map(body -> (String) body.get("target")).orElse("auto");
+    // TODO [ISW] Eclipse RCP ignores the `activate` flag passed to IWorkbenchPage.openEditor. The editor is always activated even if passing `false`.

```

Comment:
What is the expectation here? Calling openEditor and then have non-active editor? The activation flag actually relates to the enclosing Part not the Editor, which then gets activated.

Reply:
The SCION workbench supports opening a new view without activating it, i.e., without bringing it to the front. This is the same behavior supported by browsers when clicking a link on a website and pressing the control key.

What do you mean with the enclosing part? Do you mean the page? From what I understand of the Eclipse Workbench architecture, the editor itself is the part (IEditorPart), similar to the view (IViewPart). However, I may be wrong.

According to JavaDoc, I would have expected it to work as I described.

```java
/**
 * @param activate   if <code>true</code> the editor will be activated
 */
IEditorPart openEditor(final IEditorInput input, final String editorId, final boolean activate, final int matchFlags) throws PartInitException;
```